build.sbt

```diff
@@ -10,21 +10,34 @@ licenses := Seq(
 
 organization := "io.flow"
 
-scalaVersion := "2.13.1"
+scalaVersion := "2.13.3"
 
-crossScalaVersions := Seq("2.12.10", "2.13.1")
+crossScalaVersions := Seq("2.12.10", "2.13.3")
 
 fork in Test := true
 
 libraryDependencies ++= Seq(
-  "org.mockito" %% "mockito-scala-scalatest" % "1.14.8" % Test,
+  "org.mockito" %% "mockito-scala-scalatest" % "1.15.0" % Test,
   "org.scalatest" %% "scalatest" % "3.2.1" % Test,
   "org.scala-lang.modules" %% "scala-xml" % "1.3.0",
   "org.scala-lang.modules" %% "scala-parser-combinators" % "1.1.2",
-  compilerPlugin("com.github.ghik" %% "silencer-plugin" % "1.6.0" cross CrossVersion.full),
-  "com.github.ghik" %% "silencer-lib" % "1.6.0" % Provided cross CrossVersion.full,
 )
 
+
+libraryDependencies ++= (scalaBinaryVersion.value match {
```

Comment:
Issue I encountered is that 1.7.0 of silencer is not available for 2.12.x

Reply:
Are all of the warnings gone in 2.13?

Reply:
Using nowarn annotation for 2.13.  2.12.x, scala-collection-compat adds definition of nowarn annotation, and apparently silencer compiler plugin can enforce it but it didn't work for me.  I added the silencer globalFilter for 2.12 to get around it.